modules/mobile/spec/request/appointments_list_request_spec.rb

```diff
@@ -296,31 +317,48 @@
         let(:cc_appointment_with_provider) { response.parsed_body['data'][2]['attributes'] }
         let(:covid_appointment) { response.parsed_body['data'][6]['attributes'] }
 
+        it 'date range defaults to the beginning of the previous year to one year from now' do
```

Comment:
If you choose to follow my suggestion above to move these tests into a describe, this can go away.